config/rbac/role.yaml

```diff
@@ -29,17 +29,6 @@ rules:
   - patch
   - update
   - watch
-- apiGroups:
-  - kappctrl.k14s.io
```

Comment:
We should remove the kapp dependency (I think you'll see breadcrumbs in `Makefile` and the install script template).

Reply:
Cool. Thanks I will check on that one yet.

Reply:
Hi @joelanford , I finally removed kapp from Makefile, install scripts and also from the scheme package I put on the last PR. I guess that should cover it. Let me know.